src/main/java/mate/academy/model/User.java

```diff
@@ -49,4 +50,14 @@ public void setPassword(String password) {
     public void setSalt(byte[] salt) {
         this.salt = salt;
     }
+
+    @Override
+    public String toString() {
+        return "User{"
+                + "id=" + id
+                + ", email='" + email + '\''
+                + ", password='" + password + '\''
+                + ", salt=" + Arrays.toString(salt)
```

Comment:
It's not safe to add password and salt fields in toString() method. Lets remove them